emuu-client/src/components/UserAuthentication/newRegister.js

```diff
@@ -66,30 +123,35 @@ function Register() {
           <Box component="form" sx={{ mt: 3 }}>
             <Grid container spacing={0}>
               <Grid item xs={12}>
-                <TextField
+                <input
                   required
                   fullWidth
                   id="username"
                   label="Username"
                   name="username"
                   autoComplete="username"
                   value={userName}
+                  className="register-input"
+                  placeholder="Username"
                   onChange={(e) => setUserName(e.target.value)}
                 />
                 <Grid item xs={12}>
-                  <TextField
+                  <input
                     required
                     fullWidth
                     id="email"
                     label="Email Address"
                     name="email"
                     autoComplete="email"
                     value={email}
-                    onChange={(e) => setEmail(e.target.value)}
+                    className="register-input"
+                    placeholder="Email Address"
+                    onChange={(e) =>{  document.querySelector("#password-err").style.color = "black";document.querySelector("#password-err").innerHTML ="<small> Password must be at least 8 characters with 1 special character and 1 uppercase character </small>";setEmail(e.target.value)}}
```

Comment:
Why is there a password error for email? Shouldn't there be an email error message?